.mci.yml

```diff
@@ -371,6 +400,75 @@ functions:
                   # logic to handle ${test_params}.
                   MONGOC_VERSION=${mongoc_version|master} .evergreen/build_example_projects.sh
 
+                  if [ $(echo ${test_params} | grep -c UBSAN_OPTIONS) -ge 1 ]; then
```

Comment:
Doubling the C driver compile time and the test time for each task seems like a lot to check for errors due to alignment. Can we create a separate `compile_and_test_without_alignment` task? We can add that to each variant, but it can at least run alongside other tasks instead of adding onto existing tasks.

Reply:
Excellent point.  I reworked the change and it looks much nicer now.  Adding a separate task allows for much better reuse of the Evergreen functions we already have.  The whole diff is much easier to read and much simpler as well.